google-account-plugin/src/com/google/cloud/tools/intellij/login/ui/GoogleLoginAction.java

```diff
@@ -69,15 +65,14 @@ public void update(AnActionEvent e) {
       if (image == null) {
         presentation.setIcon(GoogleLoginIcons.DEFAULT_USER_AVATAR);
```

Comment:
For the default icons, we have 16x16 and 32x32 already, the latter is for retina displays. The scaling we're doing is on the icon we pull from their profile, which I believe is coming in at 96x96 to support the account popup size. We just resize for the smaller icon.